ipv6/prefix.go

```diff
@@ -162,6 +162,28 @@ func (me Prefix) Uint64() (addressHigh, addressLow, maskHigh, maskLow uint64) {
 	return
 }
 
+// prefixUpperLimit returns a new Prefix with all bits after `length` set to 1s. Note
+// that this method ignores special cases where a broadcast address doesn't
+// make sense like in a host route or point-to-point prefix (/128 and /127). It
+// just does the math.
+func (me Prefix) prefixUpperLimit() Prefix {
```

Comment:
I guess we did need this afterall. You did the right thing not to export it though.